### PR TITLE
Bibdk2021 1101 alfa 0 migration af suggest fragments js

### PR DESCRIPTION
**Pull request by akri-dbcdk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Set to false in production. When set to true, the session id is set to "test", w
 - **NEXT_PUBLIC_API_URL**
 URL to the GraphQL API. Default is http://bibliotekdk-next-api-1.febib-staging.svc.cloud.dbc.dk/graphql.
 - - **NEXT_PUBLIC_FBI_API_URL**
-URL to the FBI_API GraphQL API. Default is https://fbi-api.staging.k8s.dbc.dk/default/graphsql.
+URL to the FBI_API GraphQL API. Default is https://fbi-api.dbc.dk/default/graphql.
```
Comment (pjohans) [inline README.md:31]: should default to bibdk21 .. not default

File: e2e/cypress/integration/search.spec.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 describe("Search", () => {
-  context(`Form`, () => {
```
Comment (pjohans) [inline e2e/cypress/integration/search.spec.js:2]: the context is something for the mockups - ask @kfudbc

File: src/components/header/expandedsearch/ExpandedSearch.js [hunk 3]
```diff
@@ -63,44 +59,34 @@ export function ExpandedSearch({
...
-                <Label for="advanced-search-title">
-                  {translations(workType).labelSubject}
-                </Label>
-              </div>
-              <SuggesterTemplate
-                type="subject"
-                title={translations(workType).labelSubject}
-              />
-            </div>
+            {singleSearchInputParams.map(
```
Comment (pjohans) [inline src/components/header/expandedsearch/ExpandedSearch.js:82]: :1st_place_medal:

File: e2e/cypress/integration/search.spec.js (hunk 2 above)
Comment (kfudbc) [reply e2e/cypress/integration/search.spec.js:2]: context og describe gør vist det samme

**Review by kfudbc — APPROVED** [review]
Synes det ser godt ud!

**Pull request merged**